src/server.c

```diff
@@ -711,6 +711,17 @@ void trackInstantaneousMetric(int metric, long long current_reading) {
     server.inst_metric[metric].last_sample_count = current_reading;
 }
 
+void trackInstantaneousAvgMetric(int metric, long long current_sum, long long current_cnt) {
```

Comment:
please add some comment explaining what this function does and how.
maybe by explaining the differences from the other one above.